src/search.c

```diff
@@ -263,12 +379,58 @@ static bool file_search(char *filename, char *path, char *search_term)
 	return false;
 }
 
-/* report_find()
+double guess_score(char *bundle, char *file, char *search_term)
+{
+	double multiplier = 1.0;
+	double score = 1.0;
+
+	char bnb[PATH_MAX];
+	char dnb[PATH_MAX];
+
+	char *bn;
+	char *dn;
+
+	strcpy(bnb, file);
```

Comment:
Since you are making a copy, you can just use strrchr to find the last '/' and change it into a NUL.

Reply:
why not use `dirname`? I assume it is basically doing exactly that.

Reply:
It is making 2 copies, then calling dirname and basename. My point is it could make one copy, split it, and then use the two halves. Reduces stack space, copying, scanning.

Reply:
I see what you are saying, +1